File: UaicLibraryWebSpa/src/app/models/book-administration.model.ts
import { MultiSelectItem } from '../components/common/multiselect/multi-select-item';
import { BookCategory } from './book-category';

export class BookAdministrationModel {
    constructor(
        public id: number,
        public isbn: string,
        public bookAuthors: Array<MultiSelectItem>,
        public description: string,
        public name: string,
        public numberOfPages: number,
        public bookCategoryId: number,
        public publishDate: Date,
        public facultyId: number,
        public bookUrl: string,
        public imageUrl: string,
    ) { }
}

File: UaicLibrary.Domain/BookManagement/BookAdministrationModel.cs
using System;
using System.Collections.Generic;
using UaicLibrary.Domain.GeneralModels;

namespace UaicLibrary.Domain.BookManagement
{
    public class BookAdministrationModel
    {
        public int Id { get; set; }
        public string Isbn { get; set; }
        public IList<SelectItemModel> BookAuthors { get; set; }
        public string Description { get; set; }
        public string Name { get; set; }
        public int NumberOfPages { get; set; }
        public int BookCategoryId { get; set; }
        public int FacultyId { get; set; }
        public string BookUrl { get; set; }
        public string ImageUrl { get; set; }
        public DateTime PublishDate { get; set; }
        public string BaseImagePath { get; set; }

    }
}
